CHANGELOG.md

```diff
@@ -0,0 +1,98 @@
+- Update fronts accordingly to CSV edits [#61](https://github.com/Mapotempo/optimizer-api/pull/61)
+- Ensure matrix, matrix_id and matrix_index consistency [#62](https://github.com/Mapotempo/optimizer-api/pull/62)
+- Dump filtered problems [#62](https://github.com/Mapotempo/optimizer-api/pull/62)
+- Default `multi_trips` value is now 1 [#47](https://github.com/Mapotempo/optimizer-api/pull/47)
+- Bump VROOM to v1.5.0 [#66](https://github.com/Mapotempo/optimizer-api/pull/66)
+- Reduce memory usage for scheduling problems [#71](https://github.com/Mapotempo/optimizer-api/pull/71)
+- Reduce OR-Tools computation time within scheduling heuristic [#78](https://github.com/Mapotempo/optimizer-api/pull/78)
+- Route referents [#89](https://github.com/Mapotempo/optimizer-api/pull/89)
+- Bump Ruby to 2.5.5 [#97](https://github.com/Mapotempo/optimizer-api/pull/97)
+- Delegate quantity rounding to optimizer-ortools [#98](https://github.com/Mapotempo/optimizer-api/pull/98)
+- Improve Force start & breaks through optimizer-ortools [#98](https://github.com/Mapotempo/optimizer-api/pull/98)
+
+### Removed
+
+- `solver_parameter` from the model, it now fulfills `solver` and `preprocessing_first_solution_strategy` parameters
+- OR-Tools logs within sheduling heuristic [#28](https://github.com/Mapotempo/optimizer-api/pull/28)
+- Relation id [#31](https://github.com/Mapotempo/optimizer-api/pull/31)
+- Vehicle sort by day index [#31](https://github.com/Mapotempo/optimizer-api/pull/31)
+- `overall_duration` parameter from API as it is not functional [#46](https://github.com/Mapotempo/optimizer-api/pull/46)
+
+### Fixed
+
+- Resolution time when using split_independent or max_split procedures
+- Alternative activities with empties and fills
+- Unreachable points detection
+- Shift computation for sheduling problems
+- Sheduling heuristic now rejects shipments and rests [#28](https://github.com/Mapotempo/optimizer-api/pull/28)
+- Reject empty problems [#35](https://github.com/Mapotempo/optimizer-api/pull/35)
+- Negative time limits [#35](https://github.com/Mapotempo/optimizer-api/pull/35)
+- Expand multi_trips only once [#36](https://github.com/Mapotempo/optimizer-api/pull/36)
+- Remove data from already started jobs [#38](https://github.com/Mapotempo/optimizer-api/pull/38)
+- Clean interrupted working jobs at startup [#38](https://github.com/Mapotempo/optimizer-api/pull/38)
+- Error when delete operation is invoked immedialty after get [#38](https://github.com/Mapotempo/optimizer-api/pull/38)
+- Coerce `first_solution_strategy` into array [#38](https://github.com/Mapotempo/optimizer-api/pull/38)
+- Avoid unecessary route generation [#46](https://github.com/Mapotempo/optimizer-api/pull/46)
+- Correclty handle consecutive delete operations [#52](https://github.com/Mapotempo/optimizer-api/pull/52)
+- Solution returned from cache [#53](https://github.com/Mapotempo/optimizer-api/pull/53)
```

Comment:
The bug this PR fixes has been introduced after master -- i.e., from the point of view of releases this bug never existed. 

Which reminds me that other items might have this problem but since our releases take too long, it is hard to keep track at the moment. 

At least this item, if I am not mistaken, doesn't need mentioning.

Reply:
Ok with halil if release has a commit with ON and another woth OFF